targets/TARGET_STM/TARGET_STM32G0/us_ticker_data.h

```diff
@@ -31,6 +33,18 @@
 #define TIM_MST_RESET_ON   __TIM2_FORCE_RESET()
 #define TIM_MST_RESET_OFF  __TIM2_RELEASE_RESET()
 
+#else
+
+#define TIM_MST      TIM3
+#define TIM_MST_IRQ  TIM3_IRQn
+#define TIM_MST_RCC  __TIM3_CLK_ENABLE()
+#define TIM_MST_DBGMCU_FREEZE  __HAL_DBGMCU_FREEZE_TIM3()
+
+#define TIM_MST_RESET_ON   __TIM3_FORCE_RESET()
+#define TIM_MST_RESET_OFF  __TIM3_RELEASE_RESET()
+
+#endif
+
 #define TIM_MST_BIT_WIDTH  32 // 16 or 32
```

Comment:
I see from the datasheet that TIM3 is 16-bit. Is my understanding correct?

Reply:
good catch.
Corrected.